cmd/otelcontribcol/go.mod

```diff
@@ -2,9 +2,9 @@
 
 module github.com/open-telemetry/opentelemetry-collector-contrib/cmd/otelcontribcol
 
-go 1.21.0
+go 1.21.9
 
-toolchain go1.21.12
+toolchain go1.22.3
```

Comment:
This shouldnt need changing as part of this function add

Reply:
not a problem will change that in grok